ragas-eval-flow/flow.dag.yaml

```diff
@@ -0,0 +1,65 @@
+id: template_eval_flow

```

Comment:
could we integrate this in CI in the same PR? Just like the other evaluation flow?

Reply:
Done.